web-test-runner.config.js

```diff
@@ -23,9 +24,21 @@ export default {
 		},
 		{
 			name: 'perf',
-			files: getPattern('perf')
+			files: getPattern('perf'),
```

Comment:
I attempted splitting this config into its own file but I couldn't get it working properly.